docs/swagger.yml

```diff
@@ -47,7 +47,7 @@ paths:
         - "Host Credential"
       summary: "Add a new host credential"
```

Comment:
need to replace every "host credential" -> "credential"

Reply:
Fixed